README.md

```diff
@@ -69,3 +69,34 @@ Follow these steps to create a new API endpoint in this codebase:
 - before the next interview, we will provide PR feedback, to give you a chance to improve your solution before the call.
 
 - feel free to ask questions, as we want to simulate working with the team.
+
+## code structure
+
+dist
+configs
+src
```

Comment:
Solid directory changes to make this scale out more. Would like to see the `random-get` files also consolidated into this new structure.